docs/arguments.md

```diff
@@ -238,6 +238,17 @@ Environment Variable: WATCHTOWER_MONITOR_ONLY
 
 Note that monitor-only can also be specified on a per-container basis with the `com.centurylinklabs.watchtower.monitor-only` label set on those containers.
 
+## With label taking precedence over environment variables
```

Comment:
Environment variables sounds a bit too specific, as it concerns both command arguments and environmental variables (where the term arguments are generally preferred, since that is what we call the section).

Reply:
fixed